runtime/src/lib.rs

```diff
@@ -242,9 +242,9 @@ parameter_types! {
 
 impl pallet_evm::Trait for Runtime {
 	type FeeCalculator = ();
-	type CallOrigin = EnsureAddressTruncated;
-	type WithdrawOrigin = EnsureAddressTruncated;
-	type AddressMapping = HashedAddressMapping<BlakeTwo256>;
+	type CallOrigin = EnsureAddressRoot<AccountId>;
```

Comment:
Shouldn't this be `type CallOrigin = EnsureAddressSame;`?

https://substrate.dev/rustdocs/v2.0.0/pallet_evm/struct.EnsureAddressSame.html

Reply:
You are right